common/components/controllers/AboutProjectController.jsx

```diff
@@ -228,29 +228,30 @@ class AboutProjectController extends React.PureComponent<{||}, State> {
     );
   }
 
-  _renderPositionsOld(): ?Array<React$Node> {
-    const project = this.state.project;
-    return project && project.project_positions && project.project_positions.map((position, i) =>
-      <div key={i}>
-        <span className="pseudo-link" onClick={this.showPositionModal.bind(this,position)}>{position.roleTag.display_name}</span>
-      </div>
-    );
-  }
-
   _renderPositions(): ?Array<React$Node> {
     const project = this.state.project;
-    return project && project.project_positions && project.project_positions.map((position, i) => {
-      const positionDisplay = position.roleTag.subcategory + ":" + position.roleTag.display_name;
-      return (
-          <div key={i}>
-          {
-            position.descriptionUrl
-            ? <a href={position.descriptionUrl} target="_blank" rel="noopener noreferrer">{positionDisplay}</a>
-            : <span>{positionDisplay}</span>
-          }
-          </div>
-        );
-    });
+    return project && project.project_positions && project.project_positions
+      .sort(function(a, b) {
```

Comment:
You should be able to do the sorting in one line using
https://lodash.com/docs/4.17.10#sortBy

Ex:
_.sortBy(project.project_positions, [function(a) { return (a.roleTag.subcategory + a.roleTag.display_name).toLowerCase(); }])